lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -650,7 +650,18 @@ def upload_students_csv(_xmodule_instance_args, _entry_id, course_id, task_input
     task_progress.update_task_state(extra_meta=current_step)
 
     # Perform the upload
-    upload_csv_to_report_store(rows, 'student_profile_info', course_id, start_date)
+    # Note: values in each row are expected to be unicode, str, int, or
```

Comment:
Actually, these values are either unicode, int, or None. Don't see why it should change.